app/views/admin/products_v3/index.html.haml

```diff
@@ -17,3 +17,5 @@
       .spinner
       = t('.loading')
   #products-content
+  #product-delete-action-modals
+  #variant-delete-action-modals
```

Comment:
I forgot that we render all data in a reflex. I personally don't like it but that's unrelated to this PR.